CMAKE 4.0 removed compatibility with CMAKE < 3.5

### Expected behavior

Successful compilation with CMAKE 4.0

### Actual behavior

CMAKE removed with their newest release (4.0.0) compatibility with CMAKE < 3.5.
I can override this of couse, but best would addressed upstream. Especially if the compatibility was/is necessary.

### Steps to reproduce

Compile the source with CMAKE 4.0

```
CMake Error at CMakeLists.txt:5 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
```

### GVM versions

**gsa:** (gsad --version)

**gvm:** (gvmd --version)

**openvas:** (openvas --version)

**gvm-libs:** 22.20.0

Apart from `gvm-libs` this affects all the others where the minimum version is set to 3.0

### Environment

**Operating system:**

Linux

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: GPL-2.0-or-later
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5)
 
 message ("-- Configuring the Greenbone Vulnerability Management Libraries...")
 

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -12,7 +12,7 @@ some supported platforms.
 General build environment:
 
 * a C compiler (e.g. gcc)
-* cmake >= 3.0
+* cmake >= 3.5
 * pkg-config
 
 Specific development libraries:

